fedoracommunity/widgets/package/bugs.py

```diff
@@ -43,23 +43,27 @@ def prepare(self):
 
         releases = []
         self.filters = {'package': self.package}
-        # This is getting the list of "collections", which really correpond to releases.
-        # You can *probably* get that info from Bodhi, first.  If not, try PDC.
-        pkgdb = get_connector('pkgdb')
-        collections = pkgdb.get_collection_table(active_only=True)
-
-        for id, collection in collections.items():
-            name = collection['name']
+        bodhi = get_connector('bodhi')
```

Comment:
There's some bug in here -- we looked at the UI and saw the difference.  No clue what the bug is yet, but you know how to reproduce it now.